pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -33,19 +38,23 @@ stdenv.mkDerivation {
   outputs = [ "out" "dev" ];
   outputBin = "dev";
 
-  configureFlags = [
-    # not auto-detected by default
-    "--with-graphite2=${if withGraphite2 then "yes" else "no"}"
-    "--with-icu=${if withIcu then "yes" else "no"}"
-    "--with-gobject=yes"
-    "--enable-introspection=yes"
-  ]
-    ++ stdenv.lib.optional withCoreText "--with-coretext=yes";
+  mesonFlags = [
+    (boolMesonFlag "graphite" withGraphite2)
+    (boolMesonFlag "icu" withIcu)
+    (boolMesonFlag "coretext" withCoreText)
+    (boolMesonFlag "gobject" true)
+    (boolMesonFlag "introspection" true)
+    (boolMesonFlag "tests" true)
```

Comment:
We default auto-features to enabled:

https://github.com/NixOS/nixpkgs/blob/7e545bd7f353692cb803d799a3a5aab98b021b4d/pkgs/development/tools/build-managers/meson/setup-hook.sh#L15

So none of these are not necessary.

Tests are even enabled by default:

https://github.com/harfbuzz/harfbuzz/blob/fcddf3b9b43e391a9ec33c823ef22827d18a1fb9/meson_options.txt#L24

Reply:
Right, removed the unnecessary ones.